server/db/init.js

```diff
@@ -27,69 +27,6 @@ bookshelf.knex.schema.hasTable('users').then((exist) => {
   }
 });
 
-bookshelf.knex.schema.hasTable('scripts').then((exist) => {
```

Comment:
I removed the extras that came from me pulling from Andi's branch a while back. All the necessary tables should be there.